features/git-sync/all_branches/no_conflict/feature_branches/without_open_changes.feature

```diff
@@ -7,6 +7,10 @@ Feature: git sync --all: syncs all feature branches without open changes
       | main     | remote           | main commit     | main_file     |
       | feature1 | local and remote | feature1 commit | feature1_file |
       | feature2 | local and remote | feature2 commit | feature2_file |
+    And I have the following tags
+      | NAME   | LOCATION |
+      | local  | local    |
+      | remote | remote   |
```

Comment:
Can we make something a bit more realistic. at least like `local-v1` or something so it stands out as a tag.